.github/scripts/upload-github.sh

```diff
@@ -0,0 +1,38 @@
+#1/bin/sh
+
+set -eo pipefail
+
+# Upload the first json to the pytorch benchmark data repository
+TODAY=$(date +"%Y%m%d")
+SUBDIR="nightly-v0"
+DATA_DIR=${HOME}/benchmark-results/${TODAY}_gh${GITHUB_RUN_ID}
+BENCHMARK_ABS_FILE=$(find ${DATA_DIR} -name *.json | head -n 1)
+# Make sure the result json exists
+if [ -z "${BENCHMARK_ABS_FILE}" ]; then
+    exit 1
+fi
+
+BENCHMARK_DATA_DIR=${HOME}/pytorch_benchmark_data
+BENCHMARK_DATA_REPO=https://github.com/wconstab/pytorch_benchmark_data
+if [ -e $BENCHMARK_DATA_DIR ]; then
+    pushd ${BENCHMARK_DATA_DIR}
+    git pull origin master
+else
+    git clone ${BENCHMARK_DATA_REPO} ${BENCHMARK_DATA_DIR}
+fi
+
+git config --global user.name "pytorchbot"
+git config --global --unset url.ssh://git@github.com.insteadof || true
+
+BENCHMARK_FILENAME=${GITHUB_SHA}_$(basename ${BENCHMARK_ABS_FILE})
+pushd ${BENCHMARK_DATA_DIR}
+mkdir -p ${SUBDIR}
+cp ${BENCHMARK_ABS_FILE} ${SUBDIR}/${BENCHMARK_FILENAME}
+ln -s -f ${SUBDIR}/${BENCHMARK_FILENAME} ${SUBDIR}/latest
+echo ${BENCHMARK_FILENAME} >> ${SUBDIR}/history
+
+git add ${SUBDIR}
+git commit -m"Add ${BENCHMARK_FILENAME} - to reproduce results, see pytorch and machine versions in .json."
+git push origin master
```

Comment:
i think it might be best to still put a check that the branch isn't a fork.

if you're sure that this script won't be run for any PR branches then it isn't necessary, but at least a comment would be good.  Explaining that git push will fail if the pytorchbot token isn't available, and it's only available to builds from the main pytorch fork.